doc/services/portability/posix/overview/index.rst

```diff
@@ -0,0 +1,131 @@
+.. _posix_overview:
+
+Overview
+########
+
+The Portable Operating System Interface (POSIX) is a family of standards specified by the
+`IEEE Computer Society`_ for maintaining compatibility between operating systems. Zephyr
+implements a subset of the standard POSIX API specified by `IEEE 1003.1-2017`_ (also known as
+POSIX-1.2017).
+
+..  figure:: posix.svg
+    :align: center
+    :alt: POSIX Support in Zephyr
+
+    POSIX support in Zephyr
+
+.. note::
+    This page does not document Zephyr's :ref:`POSIX architecture<Posix arch>`, which is used to
+    run Zephyr as a native application under the host operating system for prototyping,
+    test, and diagnostic purposes.
+
+With the POSIX support available in Zephyr, an existing POSIX conformant
+application can be ported to run on the Zephyr kernel, and therefore leverage
+Zephyr features and functionality. Additionally, a library designed to be
+POSIX conformant can be ported to Zephyr kernel based applications with no changes.
+
+The POSIX API is an increasingly popular OSAL (operating system abstraction layer) for IoT and
+embedded applications, as can be seen in Zephyr, AWS:FreeRTOS, TI-RTOS, and NuttX.
+
+Benefits of POSIX support in Zephyr include:
+
+- Offering a familiar API to non-embedded programmers, especially from Linux
+- Enabling reuse (portability) of existing libraries based on POSIX APIs
+- Providing an efficient API subset appropriate for small (MCU) embedded systems
+
+.. _posix_subprofiles:
+
+POSIX Subprofiles
+=================
+
+While Zephyr supports running multiple `threads <threads_v2>`_ (possibly in an `SMP <smp_arch>`_
+configuration), as well as `Virtual Memory and MMUs <x86_developer_guide>`_, Zephyr code and data
```

Comment:
Not so sure about linking to an x86 specific page? But not sure if there is a better option either (btw thanks very much for the effort in adding plenty of cross references!).
Would hardware/porting/arch.rst or kernel/usermode/memory_domain.rst be any better?

Reply:
Will take a look

Reply:
Hmm... so the memory management API exists
https://docs.zephyrproject.org/apidoc/latest/group__mm__drv__apis.html
but there is nothing other than API docs for it. I also suspect it isn't super widely used outside of Intel.

Reply:
Suggested adding a Virtual Memory item to docs in #64049.